src/functions/template-tags/google-map.php

```diff
@@ -117,30 +117,35 @@ function tribe_get_embedded_map( $post_id = null, $width = null, $height = null,
 }
 
 /**
- * Google Map Embed Test
- *
- * Check if embed google map is enabled for this event (or venue ).
+ * Check if the "Show Map" option is enabled for the event or venue.
  *
  * @category Events
```

Comment:
Should category include Venues?

Reply:
Like we discussed on slack, `@category` is deprecated and we're going to leave them as they are for now. :meh: